### PR TITLE
Adding TCP Info logging support to header rewrite

### PR DESCRIPTION
**Pull request by dbhat**
This change to the header rewrite plugin allows TCP Info logs to be written out as part of the header.

### CODE REVIEW

**Comment by vmamidi** [comment]
@dbhat Can you also update the documentation?

**Comment by masaori335** [comment]
[approve ci]
[add to whitelist]

File: doc/admin-guide/plugins/header_rewrite.en.rst [hunk 1]
```diff
@@ -515,6 +523,7 @@ The absence of an operand for conditions which accept them simply requires that
 a value exists (e.g. the content of the header is not an empty string) for the
 condition to be considered true.
 
+
```
Comment (randall) [inline doc/admin-guide/plugins/header_rewrite.en.rst:526]: are these whitespace and `:wq` changes intentional?

Comment (dbhat) [reply doc/admin-guide/plugins/header_rewrite.en.rst]: Corrected

**Comment by maskit** [comment]
It may be nice to have a subsection about `TXN_CLOSE_HOOK` under Hook Conditions section on the documentation. If I understand correctly, the hook is special because it's too late to modify/add response headers that will be sent to a client and it only makes sense if you add internal headers.

**Comment by dbhat** [comment]
@maskit: Added to documentation in [commit](https://github.com/apache/trafficserver/pull/7441/commits/d621cee2ad994349bfac1a3d2114cc50e3169559)

**Comment by maskit** [comment]
Rerunning autest. At minimum tests for header_rewrite pass on my MBP.

[approve ci autest]

**Comment by bryancall** [comment]
I will review this.

**Pull request closed without merging**